modules/manage/pages/security/authorization/rbac.adoc

```diff
@@ -0,0 +1,247 @@
+----
+
+This diagram shows the relationships between users, roles, and ACLs, including the example `Software Eng.` role:
+
+image::shared:rbac-overview.png[RBAC multiplicity]
+
+Building off the diagram, consider the situation where you want to create a new topic, `private-software-versions`, where users self-report the version of a component they are using. If you were using the ACL authorization mechanism, you would need to assign this ACL to every user in your Redpanda installation. Using RBAC allows you to make one update and apply it to everyone with that role. Adding the write permissions for this topic to the `User` role means everyone with that role (which in our diagram is all authenticated users) gains the authorization immediately. The `rpk` command to accomplish this is:
+
+[,bash]
+----
+rpk acl create --operation write --topic private-software-versions --allow-role User
+----
+
+== Manage users and roles
+
+Redpanda allows you to manage your RBAC configurations through the use of `rpk`. The Redpanda Admin API provides another means of interacting with RBAC. Examples of using `rpk` for each RBAC operation are given here.
+
+=== Create a role
+
+Creating a new role is a two-step process. First you define the role, giving it a unique and descriptive name. Second, you assign one or more ACLs to allow or deny access for the new role. This defines the permissions that are inherited by all users assigned to the role. It is possible to have an empty role with no ACLs assigned.
+
+Administrators use `rpk` to create a new role by issuing the command:
+
+[,bash]
+----
+rpk security role create [ROLE_NAME]
+----
+
+After the role is created, administrators create new ACLs and assign this role either allow or deny permissions. For example:
+
+[,bash]
+----
+rpk acl create ... --allow-role [ROLE_NAME]
+----
+
+An example of creating a new role:
+[,bash]
+----
+$ rpk security role create red
+Successfully created role "red"
+
+ACLs can now be added to this role using
+  rpk security acl create --allow-role "RedpandaRole:red" [acl-flags]
+
+Check 'rpk security acl create --help' for more information about how to create
+an ACL.
+----
+
+=== Delete a role
+
+When a role is deleted, Redpanda carries out the following actions automatically:
+
+- All role ACLs are deleted.
+- All users assignments to the role are removed.
+
+Redpanda will list all impacted ACLs and role assignments as part of executing this command. You will receive a prompt asking you to confirm the deletion action. The delete operation is irreversible.
+
+Using `rpk`, administrators delete a role by issuing the command:
+
+[,bash]
+----
+rpk security role delete [ROLE_NAME]
+----
+
+An example of deleting a role:
+[,bash]
+----
+$ rpk security role delete red
+PERMISSIONS
+===========
+PRINCIPAL         HOST  RESOURCE-TYPE  RESOURCE-NAME  RESOURCE-PATTERN-TYPE  OPERATION  PERMISSION  ERROR
+RedpandaRole:red  *     TOPIC          asdf           LITERAL                ALL        ALLOW
```

Comment:
I remember @Feediver1 asking us to drop the usage of random strings as 'asdf' in other PRs, maybe rename this example to `books` or something else :sweat_smile:

Reply:
I scrubbed all the examples, thanks for calling that out!